packages/dds/tree/src/tree/cursor.ts

```diff
@@ -210,7 +211,7 @@ export const enum CursorLocationType {
     Fields,
 }
 
-export interface ITreeCursorSynchronous extends ITreeCursor{
+export interface ITreeCursorSynchronous extends ITreeCursor {
```

Comment:
I think this should have a doc comment, but I guess it'd be a pretty repetitive one so I'll leave that up to you.

Reply:
```suggestion
/**
 * {@link ITreeCursor} that is never pending.
 */
export interface ITreeCursorSynchronous extends ITreeCursor {
```